src/components/molecules/AdminVenueCard/AdminVenueCard.tsx

```diff
@@ -30,8 +30,17 @@ export const AdminVenueCard: React.FC<AdminVenueCardProps> = ({ venue }) => (
       />
       <h3>{venue.name}</h3>
     </div>
-    <Link className="admin-venue-card__button" to={`/admin_v2/${venue.id}`}>
-      Manage Party
+    <Link
+      className="admin-venue-card__button"
+      to={`/admin_v2/venue/${venue.id}`}
```

Comment:
+1 on the idea for easier routes management, but not sure if and how it's best to go about it since parts of those routes are constants and context dependent,

Reply:
@goran-peoski-work what aspects are ‘context dependent’ in a way that you feel it wouldn’t work?

Reply:
Still have to learn the codebase, so I'm just making a strawman case here.
In context as in Admin or Login/Auth, if we consider them a modules on their own, should they come with their own routing tucked in their own directory structure or should one general utility/constants file for the entire project suffice?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/767b8397b022e9e276ed46c3f1bd384570fa4751

Reply:
@goran-peoski-work For the utilise suggested above that won’t really matter, as they are more localised to just ‘wrapping’ the url string creation as we do for other urls in utilise/url. That way we use a single semantic function call for it rather than ‘magic strings’ all over the place.

As far as routing goes, react router is designed to be used effectively with nested subroutes. We almost certainly don’t want to keep all routing at the root level in a single routes file. The nuance right now is a little arbitrary, but this will become a lot more important in future when it comes to bundle splitting, tree shaking, lazy loading, and similar optimisations.